ruby/ql/test/library-tests/ast/CONSISTENCY/AstConsistency.expected

```diff
@@ -0,0 +1,2 @@
+extractionError
+| calls/calls.rb:8:1:8:2 | A parse error occurred. Check the syntax of the file. If the file is invalid, correct the error or exclude the file from analysis. |
```

Comment:
I think the expression on line 8 is just nonsensical, probably best remove it unless the `ruby` interpreter accepts it.

Reply:
Thanks; fixed now.